server/middlewares/multipart.ts

```diff
@@ -0,0 +1,41 @@
+import { Next } from "koa";
+import { bytesToHumanReadable } from "@shared/utils/files";
+import { InvalidRequestError } from "@server/errors";
+import { APIContext } from "@server/types";
+import { getFileFromRequest } from "@server/utils/koa";
+
+export default function multipart({
+  maxAllowedFileSize = 26214400,
```

Comment:
```suggestion
  maximumFileSize,
```

Rename, and lets remove this hardcoded default so that a limit has to be passed when used